docs/configuration/configuration.md

```diff
@@ -169,12 +169,16 @@ basic_auth:
   [ password_file: <string> ]
 
 # Sets the `Authorization` header on every scrape request with
-# the configured bearer token. It is mutually exclusive with `bearer_token_file`.
-[ bearer_token: <secret> ]
-
-# Sets the `Authorization` header on every scrape request with the bearer token
-# read from the configured file. It is mutually exclusive with `bearer_token`.
-[ bearer_token_file: <filename> ]
+# the configured credentials.
+authorization:
+  # Sets the authentication type of the request.
```

Comment:
s/authentication/authorization/, for consistency. Arguably, authentication is more correct here, but HTTP calls it authentication type.

Reply:
We often use authentication below. It's kind of confusing, now that we have a section explicitly called `authorization`.

Reply:
Well I know that but:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization

They mix authorization but say authentication type and authentication credentials...

Reply:
:facepalm: yeah, if even they cannot be consistent… then perhaps always call it authentication, unless we are literally referring to "the `Authorization` header" or something. (Perhaps that's what we are doing already?) Whatever, just wanted to raise the issue so that you deal with it as you see fit.

Reply:
I have changed authorization by authentication in https://github.com/prometheus/common/pull/275